packages/metals-languageclient/src/getJavaHome.ts

```diff
@@ -52,6 +54,26 @@ async function validateJavaVersion(
   return false;
 }
 
+export async function fromPath(
+  javaVersion: JavaVersion,
+  outputChannel: OutputChannel
+): Promise<string | undefined> {
+  let javaExecutable = findOnPath(["java"]);
+  if (javaExecutable) {
+    let realJavaPath = realpathSync(javaExecutable);
+    outputChannel.appendLine(
+      `Found java executable under ${javaExecutable} that resolves to ${realJavaPath}`
+    );
+    let possibleJavaHome = path.dirname(path.dirname(realJavaPath));
```

Comment:
```suggestion
    const possibleJavaHome = path.dirname(path.dirname(realJavaPath));
```
I don't think there is any reason for `let` here (also line 61 and 63).